scripts/rosdistro_freeze_source

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
```

Comment:
Is there a preferred copyright block for third-party contributions to OSRF packages?